### ISSUE TITLE
Upgrade gitlab.alloy.finos.org to GitLab EE 12.8 [was: GitHub SSO API deprecation notice]

### ISSUE THREAD

**Issue by maoo** [finos/alloy#68]
We are getting these emails everytime someone logs in:

```
Hi @finos-admin,

On February 7th, 2020 at 13:38 (UTC) your application (Alloy) used an access token (with the User-Agent Faraday v0.15.4) as part of a query parameter to access an endpoint through the GitHub API:

https://api.github.com/user

Please use the Authorization HTTP header instead as using the `access_token` query parameter is deprecated.

Depending on your API usage, we'll be sending you this email reminder once every 3 days for each token and User-Agent used in API calls made on your behalf.
Just one URL that was accessed with a token and User-Agent combination will be listed in the email reminder, not all.

Visit https://developer.github.com/changes/2019-11-05-deprecated-passwords-and-authorizations-api/#authenticating-using-query-parameters for more information.

Thanks,
The GitHub Team
```

We opened a ticket to GitLab Support; there's also an issue on https://gitlab.com/gitlab-org/gitlab/issues/202026 (please vote) /CC @gs-bracej @mcleo-d @brooklynrob

**Comment by maoo** [comment]
The APIs will stay untouched until July 1st, and there is already a fix available, so it's just a matter for GitLab to ship a product update (and for us to install it). No risk for the Alloy pilot!

See https://gitlab.com/gitlab-org/gitlab/issues/202026 for more info.

**Comment by maoo** [comment]
Fixed on 12.8 , see https://gitlab.com/gitlab-org/gitlab/issues/202026#note_298808966 .

We're in 12.7 now, I'll proceed with the update, when time allows.

@gs-bracej - we will have to schedule a 4 hours maintenance window, just to be safe.

Let's use this issue to track progress, I'll own the task.

**Comment by maoo** [comment]
Upgraded to 12.10.1 !
